src/_incydr_sdk/watchlists/client.py

```diff
@@ -28,18 +36,646 @@ class WatchlistsClient:
     def __init__(self, parent):
         self._parent = parent
         self._v1 = None
+        self._v2 = None
 
     @property
     def v1(self):
         if self._v1 is None:
             self._v1 = WatchlistsV1(self._parent)
         return self._v1
 
+    @property
+    def v2(self):
+        if self._v2 is None:
+            self._v2 = WatchlistsV2(self._parent)
+        return self._v2
+
+
+class WatchlistsV2:
```

Comment:
minor but this files getting pretty long, it might be nice to split every class into its own file

Reply:
I decided to move the v1 client out into its own file. That way when we deprecate it we can simply delete that file.